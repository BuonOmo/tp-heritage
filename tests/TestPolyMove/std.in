PC pol 0 0 0 100 50 50
MOVE pol 10 10
LIST
HIT pol 10 10
HIT pol 10 110
HIT pol 60 60
HIT pol 35 35
HIT pol 35 85
HIT pol 10 60
HIT pol 35 40
HIT pol 30 56
HIT pol 36 35
HIT pol 35 86
HIT pol 10 9
HIT pol 11 110
HIT pol 10 111
HIT pol 66 60
HIT pol 13 12
MOVE pol -10 -10
LIST
HIT pol 0 0
HIT pol 0 100
HIT pol 50 50
HIT pol 25 25
HIT pol 25 75
HIT pol 0 50
HIT pol 25 30
HIT pol 20 46
HIT pol 26 25
HIT pol 25 76
HIT pol 0 -1
HIT pol 1 100
HIT pol 0 101
HIT pol 56 50
HIT pol 3 2
EXIT
