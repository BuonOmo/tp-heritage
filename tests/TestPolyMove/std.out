OK
OK
PC pol 3 10 10 10 110 60 60
YES
YES
YES
YES
YES
YES
YES
YES
NO
NO
NO
NO
NO
NO
NO
OK
PC pol 3 0 0 0 100 50 50
YES
YES
YES
YES
YES
YES
YES
YES
NO
NO
NO
NO
NO
NO
NO
